### ISSUE TITLE
Add new TODO Guide: Commercial Open SourceEcosystem

### ISSUE THREAD

**Issue by anajsana** [todogroup/todogroup.org#227]
The initial draft can be found here: https://docs.google.com/document/d/1PjIaHnAylGO87KD4wmzZsw5J8787YMXBL9eMcUT5BmI/edit?usp=sharing

**Closing conditions**
- [x] Create PR

**Comment by anajsana** [comment]
@caniszczyk  I've added some comments to the doc to see if we can create a PR from that: https://docs.google.com/document/d/1PjIaHnAylGO87KD4wmzZsw5J8787YMXBL9eMcUT5BmI/edit?usp=sharing

Here are the 2 main challenges I've found:

1. This guide has some parts that are outdated (I've seen comments from 2017!). I've tried to delete the graphs that were out of date and update people's affiliations.
2. I've added some context at the very beginning to mention something about OSPOs and their role when creating these OS ecosystems. Is never mentioned within the text and I think it need to be clear :)

My suggestion: Accept the changes and create the PR where people can make changes and update a bit more this guide (happy to do that once this is approved) :+1:
